src/Microsoft.Sbom.Adapters/Adapters/ComponentDetection/NuGetComponentExtensions.cs

```diff
@@ -25,10 +25,10 @@ public static SbomPackage ToSbomPackage(this NuGetComponent nuGetComponent, Exte
         PackageName = nuGetComponent.Name,
         PackageVersion = nuGetComponent.Version,
         Supplier = nuGetComponent.Authors?.Any() == true ? $"Organization: {nuGetComponent.Authors.First()}" : component.Supplier,
```

Comment:
Personal preference, but when predicates and assignments start getting longer like this I tend to prefer putting them in their own variable first since it makes it easier to tell what's going on in the debugger.